README.md

```diff
@@ -654,6 +654,15 @@ You can even create custom collectors in your project by implementing the `Qossm
 As soon as an unknown collector is referenced in the config file deptrac will try to load the class in your project.
 With this you can create collectors specific for your usecase. And more people can use these custom collectors per default if you contribute them back to deptrac!
 
+### Ignoring `use` statements for namespaces
```

Comment:
Please also add this to the toc links at the top.

Reply:
https://github.com/qossmic/deptrac/pull/571/commits/c2bea8bb4745b5c8f5e5c6521eaf47a5cded7eae